pipeline/projects/__init__.py

```diff
@@ -292,11 +277,13 @@ def active_args(self, data:dict, name:str):
 	def add_props(self, data:dict, role=None, **kwargs):
 		role = role if role else 'person'
 		auth_name = data.get('auth_name', '')
+		generic_name = data.get('generic_name', '')
+		century_active = data.get('century_active', '')
 		period_match = self.anon_period_re.match(auth_name)
 		nationalities = []
 		if 'nationality' in data:
 			nationality = data['nationality']
-			if isinstance(nationality, str):
+			if isinstance(nationality, str) and nationality != 'Unknown':
```

Comment:
Is this addition part of fixing AR-170? Or was it required to get AR-170 to pass tests for some reason?